common/beerocks/bwl/dummy/nl80211_client_dummy.h

```diff
@@ -31,6 +31,18 @@ class nl80211_client_dummy : public nl80211_client {
      */
     virtual ~nl80211_client_dummy();
 
+    /**
+     * @brief Gets radio information.
+     *
+     * Radio information contains HT/VHT capabilities and the list of supported channels.
+     *
+     * @param[in] interface_name Interface name, either radio or Virtual AP (VAP).
+     * @param[out] radio_info Radio information.
+     *
+     * @return True on success and false otherwise.
+     */
+    virtual bool get_radio_info(const std::string &interface_name, sRadioInfo &radio_info) override;
```

Comment:
No need for virtual when there is already override (nitpick)

Reply:
Using `virtual` is an old habit from the times before C++11, when there was no `override`. It wasn't necessary either, but it was considered good practice to advertise programmers that a method was `virtual` without having to check the base class definition. I've also heard that some compilers might complain if `virtual` is not used in derived class, but that's not our case. 

Since `virtual` is being used together with `override` all over the code and it doesn't hurt, I'm going to leave it here as it is now. If we ever vote it in #democracy, I'll say do not use `virtual` if there is `override`. But I'll also say the use of `override` should be mandatory to avoid pitfalls.